src/CommonRLInterface.jl

```diff
@@ -58,4 +75,7 @@ reset!(env::RLBaseEnv) = CRL.reset!(env.env)
 (env::RLBaseEnv)(a) = env.r = CRL.act!(env.env, a)
 Base.copy(env::CommonRLEnv) = RLBaseEnv(CRL.clone(env.env), env.r)
 
-ActionStyle(env::RLBaseEnv) = CRL.provided(CRL.valid_actions, env.env) ? FullActionSet() : MinimalActionSet()
\ No newline at end of file
+ActionStyle(env::RLBaseEnv) = CRL.provided(CRL.valid_actions, env.env) ? FullActionSet() : MinimalActionSet()
+UtilityStyle(env::RLBaseEnv{<:CRL.AbstractZeroSumEnv}) = ZERO_SUM
```

Comment:
Should there also be something for
`UtilityStyle(env::RLBaseEnv{<:CRL.AbstractMarkovEnv})`?

Reply:
The default one is `GENERAL_SUM`. I'll add it here to avoid confusion. 👍